src/Init/NotationExtra.lean

```diff
@@ -196,6 +196,11 @@ macro_rules
     `($mods:declModifiers class $id $params* extends $[$parents:term],* $[: $ty]?
       attribute [instance] $ctor)
 
+/-- `· tac` focuses on the main goal and tries to solve it using `tac`, or else fails. -/
+syntax ("·" <|> ".") ppHardSpace many1Indent(group(tactic ";"? ppLine)) : tactic
```

Comment:
Could/should we change the initial `ppLine` in `tactiSeqIndented` to `ppHardlineUnlessUngrouped` so that callers can opt out of it?

Reply:
Well, that really only makes sense for when the syntax prefix is exactly the indentation size, so further use cases don't seem all that likely.